test/apiv2/20-containers.at

```diff
@@ -97,7 +97,7 @@ t GET    libpod/containers/${cid}/json 200 \
 t DELETE libpod/containers/$cid 204
 
 CNAME=myfoo
-podman run --name $CNAME $IMAGE -td top
+podman run -d --name $CNAME $IMAGE top
```

Comment:
This command was failing outright. I have no idea how the tests below were working at all... but I just don't have the energy to look deeper. Anyhow, they're still passing, so I'm going to stick my head in the sand.

Reply:
Well the container was successfully created and the tests below do not care if the container was running or exited immediately due an invalid command.